lib/vyasa_web/components/contexts/components.ex

```diff
@@ -29,6 +29,7 @@ defmodule VyasaWeb.Context.Components do
           phx-click={JS.push("toggle_marks_display_collapsibility", value: %{value: ""})}
           phx-target={@marks_target}
           class="flex items-center w-full hover:bg-brand-light hover:text-brand"
+          type="button"
```

Comment:
NOTE to self: 
without this, this particular click gets treated as a submit button click and unexpectedly submits the form